.github/workflows/main.yml

```diff
@@ -5,6 +5,11 @@ on:
   push:
     branches:
       - '**'
+    paths-ignore:
```

Comment:
This changes were only added to `push` events but not to `pull_request` events. That seems unlikely to be expected.

Reply:
Good catch. Added for PRs as well.